crates/astria-sequencer/src/app/tests_app/mempool.rs

```diff
@@ -147,24 +147,23 @@ async fn do_not_trigger_cleaning() {
     app.prepare_commit(storage.clone()).await.unwrap();
     app.commit(storage.clone()).await;
 
-    let alice = get_alice_signing_key();
-
     // create tx which will fail execution and not trigger flag
     // (wrong sudo signer)
-    let tx_fail = UnsignedTransaction {
-        params: TransactionParams::builder()
-            .nonce(0)
-            .chain_id("test")
-            .build(),
-        actions: vec![
+    let tx_fail = UnsignedTransaction::new(
+        vec![
             FeeChangeAction {
                 fee_change: FeeChange::TransferBaseFee,
                 new_value: 10,
             }
             .into(),
         ],
-    }
-    .into_signed(&alice);
+        TransactionParams::builder()
+            .nonce(0)
+            .chain_id("test")
+            .build(),
+    )
+    .expect("failed to build unsigned transaction")
```

Comment:
As above: this does not provide info as to why you expect this to have worked. If you think it's trivially clear why this should have worked, prefer `unwrap()`.